index.js

```diff
@@ -23,8 +30,13 @@ module.exports = {
   server: {
     createServer,
     jsonToProtobufHandlerWrapper,
+    AcknowledgingWritable,
     error: {
       wrapInErrorHandlerFactory,
     },
   },
+  utils: {
+    isObject,
+    convertObjectToMetadata,
```

Comment:
I feel like utils it something not related to project logic. Common stuff that you can use in any project. But `convertObjectToMetadata` definitely about GRPC. What do you think?